src/ecs/resources/input.rs

```diff
@@ -28,7 +41,7 @@ impl<'a> Iterator for PressedKeys<'a> {
 
 /// An iterator over the currently pressed down mouse buttons.
 pub struct PressedMouseButtons<'a> {
-    iterator: Keys<'a, MouseButton, KeyQueryState>,
+    iterator: Iter<'a, MouseButton>,
```

Comment:
is there any benefit over doing just `type PressedMouseButtons<'a> = Iter<'a, MouseButton>`?